blog/2023/09/schema-management-disaster-recovery-in-klaw.md

```diff
@@ -50,8 +50,8 @@ registered schemas, as determined by the consumer's settings.
 
 ### Schema id
 
-In Kafka events, you can create schemas for both keys and values, allowing the generation of key schemas and value
-schemas. When a schema is registered, it is automatically stored in the default `_schemas` Kafka topic and is assigned
+In Apache Kafka events, you can create schemas for both keys and values, allowing the generation of key schemas and value
+schemas. When a schema is registered, it is automatically stored in the default `_schemas` Apache Kafka topic and is assigned
 a unique identifier.
```

Comment:
Also suggest changing this to as follows: 
In Apache Kafka events, you can create schemas for both keys and values. This allows for the generation of key schemas and value schemas. Once a schema is registered, it is automatically stored in the default ` _schemas` topic of Apache Kafka and is assigned a unique identifier.